pkg/cli/admin/inspect/pod.go

```diff
@@ -126,6 +127,18 @@ func (o *InspectOptions) gatherContainerRotatedLogFiles(destDir string, pod *cor
 		return err
 	}
 
+	// when sinceTime is given we use that to compare the log file name with
+	// the provided date.
+	// when since is given we subtract the given duration from time.Now(),
+	// then use that as the requested time.
+	requestedTime := time.Date(1970, time.January, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
nit: What about? and check that `requestedTime.IsZero()` in rotated log filtering below?

```suggestion
	var requestedTime time.Time
	if len(o.sinceTime) > 0 {
	       // when sinceTime is given we use that to compare the log file name with
	        // the provided date.
		requestedTime = o.sinceTimestamp.Time
	}
	if o.since != 0 {
		// when since is given we subtract the given duration from time.Now(),
	        // then use that as the requested time.
		requestedTime = time.Now().Add(-o.since)
	}
```

Reply:
that is indeed more elegant, thanks - updated.